Bio/SeqUtils/MeltingTemp.py

```diff
@@ -716,8 +716,8 @@ def Tm_GC(seq, check=True, strict=True, valueset=7, userset=None, Na=50, K=0,
         sum(map(seq.count, ('B', 'V'))) * 66.67 / len(seq) + \
         sum(map(seq.count, ('D', 'H'))) * 33.33 / len(seq)
     if strict and tmp:
-        raise ValueError('ambiguous bases B, D, H, K, M, N, R, V, Y not ' +
-                         'allowed when strict=True')
+        raise ValueError('ambiguous bases B, D, H, K, M, N, R, V, Y not '
+                         'allowed when \'strict=True\'')
```

Comment:
Maybe use the double-quote for the string limits, to avoid escaping the single quotes you've added?

Reply:
I used the escaped single quotes elsewhere in the same code, so I stayed with them. No problem to replace them all.

Reply:
If you were following the existing module's style, that's fine then. Leave it.